src/locale/it-IT.h

```diff
@@ -0,0 +1,176 @@
+// Copyright 2020 - Enrico Gueli (@egueli)
+// Locale/language file for Italian.
+// This file will override the default values located in `defaults.h`.
+
+#ifndef LOCALE_IT_IT_H_
+#define LOCALE_IT_IT_H_
+
+#define D_STR_UNKNOWN "SCONOSCIUTO"
+#define D_STR_PROTOCOL "Protocollo"
+#define D_STR_POWER "Accensione"
+#define D_STR_ON "Acceso"
+#define D_STR_OFF "Spento"
+#define D_STR_MODE "Modalità"
+#define D_STR_TOGGLE "Alterna"
+#define D_STR_TURBO "Turbo"
```

Comment:
FYI, where the word/spelling doesn't change, you don't need to include it.
I'll remove anything obvious that doesn't change.